src/model/model.ts

```diff
@@ -1102,6 +1102,14 @@ class ScreenLines {
         let origCmd = this.cmds[cmd.lineid];
         if (origCmd != null) {
             origCmd.setCmd(cmd);
+        } else {
```

Comment:
This code handles a race condition where sometimes a copy file local to local finishes so fast that update cmd comes over the websocket before the add line update is returned, so when it tries to update a given line id, the line isn't there yet 

what I do is if the line is not found, I async wait 100 ms then try again, and in practice after that time the add line update has been processed so the line is there after the wait